bundles/org.eclipse.equinox.preferences/src/org/eclipse/core/internal/preferences/Activator.java

```diff
@@ -59,6 +62,8 @@ public class Activator implements BundleActivator, ServiceTrackerCustomizer<Obje
 	 */
 	private ServiceRegistration<org.osgi.service.prefs.PreferencesService> osgiPreferencesService;
 
+	private ServiceTracker locationTracker;
```

Comment:
You should add generics here to avoid new raw types warnings.

Reply:
Done